packages/apidom-ls/src/config/openapi/operation/documentation.ts

```diff
@@ -0,0 +1,51 @@
+
+const documentation = [
+  {
+    target: 'tags',
+    docs: '#### [`string`]\n\nA list of tags for API documentation control. Tags can be used for logical grouping of operations by resources or any other qualifier.',
+  },
+  {
+    target: 'summary',
+    docs: 'A short summary of what the operation does.',
+  },
+  {
+    target: 'description',
+    docs: 'A verbose explanation of the operation behavior. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.',
+  },
```

Comment:
@char0n  I can make this change, however, I see that the docs are already picked up from the meta doc:

![apidom-ls-pr-1946-comment-operation-externalDocs](https://user-images.githubusercontent.com/12902658/188944961-29c33c29-dc13-4b55-8459-e682d0dd7ca7.png)

Reply:
Right that's a good point. We have two options here.

### Make it explicit

As per my suggestion. I bet if somebody else will come to this file, he will think that the documentation rule is missing and will try to add it (as me just did).

### Make a comment where field is expected

Instead of defining field we leave the comment to explain how things work so that nobody tries to add the rules later.


---

Personally I would go for - **Make it explicit**. It will be consistent with current AsyncAPI rules. All the fields will be explicit and we don't always need to consult specification document  to find out what is the full enumeration of the fields.  And yes, there will be duplicate documentation strings, but in future we want to eliminate this duplicates by changing documentation rules shape from list to object, which will make ti possible to reference it easily.

Reply:
**Update**: I thought about it again in context of OpenAPI specification after studying what I did in AsyncAPI rules. There are actually rules that governs this.

For example use of `External Documentation Object` within the spec:

![image](https://user-images.githubusercontent.com/193286/189086124-5014ea64-26e6-485a-8b15-5bad960689ca.png)
![image](https://user-images.githubusercontent.com/193286/189086258-c62d4c25-d2c3-4800-a6fe-0501762c2f3f.png)
![image](https://user-images.githubusercontent.com/193286/189086462-c0ec06be-3b07-4a37-90c3-edd60e50687c.png)
![image](https://user-images.githubusercontent.com/193286/189086517-d2ada020-dcf0-43d7-a858-051548833a4d.png)

We can see that the `externalDocs` field description doesn't tell us anything significant. So we can omit the the field and let meta docs show us full documentation for `External Documentation Object`.

If the description would contain some significant information, then what we do is create a link to documentation object and provide a significant description

```js
{
  target: 'externalDocs',
  docs: '[External Documentation Object](link#externalDocumentation)\n\nSignificant description',
}
```

The same applies for example for `Path Item Object` and it's `get`, `post`, `put`, etc... fields. Their description doesn't tell us anything significant, so it's again OK to omit them.

---

To sum it up, for OpenAPI spec it makes sense omit the documentation rules for some fields, but we should put a placeholder comment there instead as suggested by **Make a comment where field is expected**.

Reply:
Done: **Make a comment where field is expected**.